docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -148,3 +169,10 @@ userStatus="$(docker exec -u root -it wmagent sh -c "passwd -S $wmaUser" | awk '
 if [ "${userStatus:0:1}" == "P" ]; then
     docker exec -u root -it wmagent sh -c "echo $wmaUser:$wmaUser | chpasswd"
 fi
+
+# Configure s-nail to use the host s-nail mail server
+docker exec -u root -it wmagent cp /etc/s-nail_host.rc /etc/s-nail.rc
```

Comment:
I was thinking that we could keep the modified s-nail.rc under `$HOST_MOUNT_DIR/admin/etc/` and simply mount it from there to `/etc/s-nail.rc`. This way we could do some of this tweaking only once in a lifetime of an agent.

The `update-alternatives` seem to be still required though - but perhaps it could go to the wmagent Dockerfile?

Reply:
That's a good point! Thank you for this clarification, Kenyi.